sandbox.sh

```diff
@@ -189,11 +190,33 @@ start_deku_cluster() {
     deku-cli sign-block "$data_directory/$i" $HASH
   done
 
+}
+
+wait_for_servers() {
   for PID in ${SERVERS[@]}; do
     wait $PID
   done
 }
 
+assert_deku_state() {
+  contract=$(cat "$data_directory/0/tezos.json" | jq '.consensus_contract' | xargs)
```

Comment:
We don't list `jq` as a dev dependency, so I'd either add it to the README or add something as:
```
if ! command -v jq
then
    echo Command jq not found, but it is required to verify the result of the test.
    exit 1
fi
```

Reply:
addressed in latest commit

Reply:
For nix only, no? By the way, how did it work before with Nix? I believe you have to list all the dependencies with nix files, don't you?

Reply:
So, when using `nix develop`, the default is for your shell environment to "come with you" - you can use any tools you already had installed, in addition to the tools nix adds (this can be overridden with the `--ignore-environment` flag).

Being used in the smoke test, `jq` is a dev dependency, not needed for consumers of the Deku who want to merely build and run it. Hence I don't think it needs a spot in the README. For Deku devs, I think the error message you added is sufficient.